src/core/Akka.Tests.Shared.Internals/AkkaSpec.cs

```diff
@@ -115,9 +115,12 @@ private static string GetCallerName()
         public static Config AkkaSpecConfig { get { return _akkaSpecConfig; } }
 
         protected T ExpectMsgPf<T>(TimeSpan? timeout, string hint, Func<object, T> function)
+            => ExpectMsgPf(timeout, hint, this, function);
+        
+        protected T ExpectMsgPf<T>(TimeSpan? timeout, string hint, TestKitBase probe, Func<object, T> function)
```

Comment:
Added ExpectMsgPf overload that can take a test probe as parameter

Reply:
Nice